[3.14.3] nameConverter.js exception

Docker image. The below error shows in the logs every bot start. Wasn't showing in the previous version.

```
[02-03-2025 @ 05:29:31.312 GMT] Loaded module: commands/streamCommand.js (Enabled)
[02-03-2025 @ 05:29:31.342 GMT] Loaded module: commands/dualstreamCommand.js (Enabled)
[02-03-2025 @ 05:29:31.347 GMT] Loaded module: commands/topCommand.js (Enabled)
[02-03-2025 @ 05:29:31.358 GMT] [ERROR] [init.js:86] Exception [TypeError: Cannot set property "changeName" of undefined to "
function changeName(oldName, newName) {
    if (oldName === undefined || oldName === null || newName === undefined || newName === null) {
        return 0;
    }
    let changed = 0, i;
    oldName = $.jsString(oldName).toLowerCase();
    newName = $.jsString(newName);
    let newNameL = $.jsString(newName).toLowerCase();
    for (i in tables) {
        let entry = $.optIniDbString(tables[i], oldName);
        if (entry.isPresent()) {
            $.inidb.set(tables[i], newNameL, entry.get());
            $.inidb.del(tables[i], oldName);
            changed++;

}
    }
    let keys = $.inidb.GetKeyList("quotes", ""), jsonArr;
    for (i in keys) {
        if ($.getIniDbString("quotes", keys[i]).toLowerCase().indexOf(oldName) > -1) {
            jsonArr = JSON.parse($.getIniDbString("quotes", keys[i]));
            $.inidb.set("quotes", keys[i], JSON.stringify([String(newName), String(jsonArr[1]), String(jsonArr[2]), String(jsonArr[3])]));
}
    }
    return changed;
}
"] Location[loadScript(./commands/nameConverter.js)] Stacktrace [nameConverter.js:99 > @nameConverter.js:18 > loadScript()@init.js:249 > loadScriptRecursive()@init.js:328 > loadScriptRecursive()@init.js:326 > init()@init.js:551 > init.js:933 > @init.js:24]
[02-03-2025 @ 05:29:31.361 GMT] [ERROR] [invoke0()@null:-2] java.lang.RuntimeException: Unable to printStackTrace
[02-03-2025 @ 05:29:31.378 GMT] Loaded module: commands/customCommands.js (Enabled)
[02-03-2025 @ 05:29:31.381 GMT] Loaded module: commands/highlightCommand.js (Enabled)
[02-03-2025 @ 05:29:31.385 GMT] Loaded module: commands/lastseenCommand.js (Enabled)
```

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -532,7 +532,7 @@ jobs:
       uses: actions/github-script@v7
       with:
         script: |
-          let {status: getStatus, data} = await octokit.rest.repos.getReleaseByTag({
+          let {status: getStatus, data} = await github.rest.repos.getReleaseByTag({
             owner: context.repo.owner,
             repo: context.repo.repo,
             tag: context.ref.substring(context.ref.lastIndexOf("/") + 1),
@@ -541,7 +541,7 @@ jobs:
           if (getStatus !== 200) {
             core.error("Failed to get the release..." + getStatus);
           } else {
-            let {status: updateStatus} = await octokit.rest.repos.updateRelease({
+            let {status: updateStatus} = await github.rest.repos.updateRelease({
               owner: context.repo.owner,
               repo: context.repo.repo,
               release_id: data.id,

--- a/javascript-source/commands/nameConverter.js
+++ b/javascript-source/commands/nameConverter.js
@@ -96,5 +96,6 @@
         $.registerChatCommand('./commands/nameConverter.js', 'namechange', $.PERMISSION.Admin);
     });
 
+    $.nameConverter = {};
     $.nameConverter.changeName = changeName;
 })();

